src/patterns/task-list/index.md.njk

```diff
@@ -0,0 +1,75 @@
+---
+title: Task list pages
+description:
+section: Patterns
+theme: Pages
+aliases:
+layout: layout-pane.njk
+status: Experimental
+---
+
+Task list pages help users understand:
+
+- the tasks involved in completing a transaction
+- the order they should complete tasks in
+- when they have completed tasks
+
+![Task list example](task-list-whole.png)
```

Comment:
Is this a good alt tag for the image? Could we make it slightly clearer, maybe something like 'A screenshot showing an example of the task list as used in a fictional service'?

Reply:
Fixed